waiter/src/waiter/core.clj

```diff
@@ -74,17 +73,16 @@
             [waiter.websocket :as ws]
             [waiter.work-stealing :as work-stealing])
   (:import (java.net InetAddress URI)
-           java.util.concurrent.Executors
+           (java.util.concurrent Executors)
```

Comment:
`java.util.concurrent.Executors` → `(java.util.concurrent Executors)` doesn't seem like it would preserve semantics...

Is this just unused?

Reply:
Never mind. I just didn't understand the Clojure semantics:

> The `:import` clause declares this lib’s use of `java.util.Date` and makes it available to code in this lib using its unqualified name.
https://clojure.org/reference/libs